app/src/main/res/layout/faq_fragment.xml

```diff
@@ -27,17 +24,59 @@
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/faqRecycler"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:overScrollMode="never"
+        android:layout_height="match_parent"
         android:clipToPadding="false"
-        android:paddingTop="8dp"
+        android:overScrollMode="never"
+        android:paddingTop="25dp"
         android:paddingBottom="60dp"
         app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
         app:layout_behavior="@string/appbar_scrolling_view_behavior"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toTopOf="parent"
+        tools:listitem="@layout/faq_list_item" />
+
+    <LinearLayout
```

Comment:
@ioane-bendingspoons when there are no items to show, the placeholder view is cutted if there is not enough space.
![device-2020-06-11-121655](https://user-images.githubusercontent.com/3175499/84373959-9db18380-abdd-11ea-983e-661601ea2d5e.png)

Reply:
I see several options to resolve this
1. Put this empty view as Recyclerview list item. 
(I don't like this because, empty list should be just an empty list)
2. Put empty view in NestedScrollView so that it can be scrolled. 
3. Avoid resizing activity on keyboard show. 
(So that bottom navigation does not pop-up when user is searching. 

I like the third option. 
@marcouberti what do you think?

Reply:
I like option 3 too. There is no need to resize the content of the page in this particular scenario.

Reply:
Commit 48d2f2dcef37806fe3b81d5e3953105d19ef79fc addresses this issue.